appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/ProfilePage.java

```diff
@@ -524,4 +540,190 @@ private boolean validEmailFrequency(String emailFrequencyString){
 
     return true;
   }
+
+  /**
```

Comment:
@jisqyv @wolberd I believe that whenever classes need to implement this class it is added as a private wrapper class. In this case it is added to hold the content of the catalog